src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -134,6 +134,16 @@
     <ProjectCapability Include="DataSourceWindow" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' >= '3.0')" />
   </ItemGroup>
 
+  <!-- List of xaml rule contexts for Cps:
```

Comment:
>List of xaml rule contexts for Cps: [](http://example.com/codeflow?start=7&length=35)

It would be nice to give more details as to what effect these have. I *believe* these are actually xaml schema contexts but it isn't clear from this comment.

Reply:
changed the comments in last commit.